playbooks/group_vars/webportals.yml

```diff
@@ -116,6 +118,13 @@ webportal_server_config_defaults:
   sia_api_port: 9980
   hsd_api_key: "{{ lookup('pipe','openssl rand -base64 32') }}"
 
+webportal_cluster_config_defaults:
+  cookie_hash_key: "{{ lookup('pipe','openssl rand -hex 32') }}"
```

Comment:
This can mess up cluster config until full cluster config generation and saving to LastPass (or other secures storage) is finished, because if the portal operator doesn't set these 2 values, each portal in cluster will generate different values.